src/app/Shared/Services/Settings.service.tsx

```diff
@@ -47,16 +52,36 @@ import {
 } from './Api.service';
 import { NotificationCategory } from './NotificationChannel.service';
 
+dayjs.extend(utc);
+dayjs.extend(timezone);
+dayjs.extend(advanced);
```

Comment:
Do these `extend()` calls need to be sprinkled around so many different files? It would be nice for cleanliness/maintenance reasons if this could just be done somewhere once globally.

Reply:
Ahh dayjs is said to be immutable. I think I misunderstood what they meant. Probably, I will write a custom hook for it.